### PR TITLE
Add @EricWF's clang contract branch

### PR DESCRIPTION
**Pull request by EricWF**

### CODE REVIEW

**Review by mattgodbolt — APPROVED** [review]
Looks good: thanks!

**Comment by EricWF** [comment]
You're scary fast. Thanks for the review!

**Comment by mattgodbolt** [comment]
too fast, sadly the build failed

File: build/build.sh [hunk 1]
```diff
@@ -145,6 +145,12 @@ dascandy-contracts-trunk)
     VERSION=dascandy-contracts-trunk-$(date +%Y%m%d)
     LLVM_ENABLE_RUNTIMES+=";libunwind"
     ;;
+ericwf-contracts-trunk)
+    BRANCH=contracts-nightly
+    URL=https://github.com/efcs/llvm-project
+    VERSION=ericwf-contracts-trunk-$(date +%Y%m%d)
+    LLVM_ENABLE_RUNTIMES+="libcxx;libcxxabi;libunwind"
```
Comment (mattgodbolt) [inline build/build.sh:152]: missing `;` at the front if you want to send a new PR!

Comment (EricWF) [reply build/build.sh:152]: Sorry about that. Sent.

Comment (mattgodbolt) [reply build/build.sh:152]: sorry I didn't spot it!

**Pull request merged**